definitions/variable/emissions/tag_long-lived-materials.yaml

```diff
@@ -0,0 +1,7 @@
+- Long-Lived Materials:
+    - Timber:
+        description: long-lived wood products
+    - Plastics:
+        description: long-lived plastics
+    - Other:
+        description: other long-lived materials
```

Comment:
again, timeframes eg "other long-lived materials storing carbon for at least 100 years"

Reply:
Added the same sentence as in https://github.com/IAMconsortium/common-definitions/blob/be138400d429363066c497535a1ab5af734f9042/definitions/variable/emissions/tag_carbon-utilization.yaml#L5